src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
Hi, Having a class will help developers to continue with HTML structuring without adding any extra CSS (.bx--breadcrumb-item:last-child::after) as well as this also providing less customization options from carbon design system. This will be an utility class, if user don't want to use, they can skip this.